CHANGELOG-1.1.md

```diff
@@ -1,3 +1,32 @@
+# TiDB Operator v1.1.1 Release Notes
+
+## Notable changes
+
+- Add `additionalContainers` and `additionalVolumes` fields so that Tidb-operator can support adding sidecars to `TiDB`, `TiKV`, `PD`, etc. ([#2229](https://github.com/pingcap/tidb-operator/pull/2229), [@yeya24](https://github.com/yeya24))
```

Comment:
```suggestion
- Add the `additionalContainers` and `additionalVolumes` fields so that TiDB Operator can support adding sidecars to `TiDB`, `TiKV`, `PD`, etc. ([#2229](https://github.com/pingcap/tidb-operator/pull/2229), [@yeya24](https://github.com/yeya24))
```